### ISSUE TITLE
Fails to use browser tool with prompt_toolkit

### ISSUE THREAD

**Issue by patriklaurell** [ErikBjare/gptme#351]
Fails to read urls

```
Preview
read_url('https://www.swiss.com/lhg/ch/en/o-d/cy-cy/zurich-copenhagen')

[13:40:32] ERROR    asyncio.run() cannot be called from a running event loop                                                                                                                     base.py:297
                    Traceback (most recent call last):
                      File "/Users/patriklaurell/repos/gptme/gptme/tools/base.py", line 292, in execute
                        yield from ex
                      File "/Users/patriklaurell/repos/gptme/gptme/tools/python.py", line 79, in execute_python
                        if not confirm("Execute this code?"):
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                      File "/Users/patriklaurell/repos/gptme/gptme/chat.py", line 107, in confirm_func
                        return ask_execute(msg)
                               ^^^^^^^^^^^^^^^^
                      File "/Users/patriklaurell/repos/gptme/gptme/util/ask_execute.py", line 110, in ask_execute
                        session.prompt(
                      File "/Users/patriklaurell/.local/pipx/venvs/gptme/lib/python3.12/site-packages/prompt_toolkit/shortcuts/prompt.py", line 1035, in prompt
                        return self.app.run(
                               ^^^^^^^^^^^^^
                      File "/Users/patriklaurell/.local/pipx/venvs/gptme/lib/python3.12/site-packages/prompt_toolkit/application/application.py", line 1002, in run
                        return asyncio.run(coro)
                               ^^^^^^^^^^^^^^^^^
                      File "/opt/homebrew/Cellar/python@3.12/3.12.8/Frameworks/Python.framework/Versions/3.12/lib/python3.12/asyncio/runners.py", line 190, in run
                        raise RuntimeError(
                    RuntimeError: asyncio.run() cannot be called from a running event loop
System: Error executing tool 'python': asyncio.run() cannot be called from a running event loop
/Users/patriklaurell/repos/gptme/gptme/tools/base.py:300: RuntimeWarning: coroutine 'Application.run_async' was never awaited
  yield Message("system", f"Error executing tool '{self.tool}': {e}")
```

**Comment by ErikBjare** [comment]
This seems to always happen on calls to `session.prompt` (either `ask_confirm` or `user input`) after using the browser tool (either `search` or `read_url`).

Probably something with how `prompt_toolkit` and `playwright` interact with async together.

**Comment by ErikBjare** [comment]
@gptme what do you think?

**Comment by ErikBjare** [comment]
lol, yikes. I guess the easiest thing is to isolate the browser stuff into its own process.

Good bot!

**Comment by ErikBjare** [comment]
I've implemented a fix in #353 that moves Playwright to a separate thread to isolate its event loop from prompt_toolkit.

The fix has been tested and works with all browser operations while maintaining prompt_toolkit functionality. Please test and let me know if you encounter any issues!
